data/resources/style.css

```diff
@@ -263,14 +270,21 @@ messagemedia overlay > messageindicators, messagesticker messageindicators {
 
 /* Placeholder */
 .message-entry > overlay > label {
-  margin: 0 9px;
+  margin-left: 9px;
+}
+
+.message-entry > overlay > entry image {
+  margin: 9px;
 }
 
 .message-entry > overlay > entry {
   padding: 0;
+  margin: 7px 0;
+  background: alpha(@window_fg_color, 0.1);
```

Comment:
I'm pretty sure this is the same color as the default one.

This is the one currently used: https://gitlab.gnome.org/GNOME/libadwaita/-/blob/main/src/stylesheet/widgets/_buttons.scss#L1

Reply:
Lol, I checked more and opacity, but I don't like how it looks, also I don't want to use opacity less than 0.1 because it's too much symbols

Reply:
In that case I suggest separate light and dark colors like this

![image](https://user-images.githubusercontent.com/63008755/195669960-79565be2-79f8-460f-98f2-6de0dd985d82.png)

![Screenshot from 2022-10-13 20-52-11](https://user-images.githubusercontent.com/63008755/195670019-9dea6366-ba62-43bb-85c6-48a8ca7388b0.png)

Reply:
It doesn't look bad, but I would like to try that first. Can you push the change?

Reply:
Yes (pushed), but I think it will better to have theme independent color, maybe semi-transparent black but I can't configure opacity what I want

Reply:
After trying that dark color a bit and playing with darker colors too, I think that the default one is the best for a simple reason: is makes the entry more distinct to the rest of the UI colors. This is good especially for lower quality display, where having good contrast is important. The darker version has too little contrast and merges visually to the rest of the UI.